src/Calculator/pch.h

```diff
@@ -44,3 +44,5 @@ namespace CustomPeers = CalculatorApp::Common::Automation;
 
 // Project Headers
 #include "App.xaml.h"
+
+#include <string>
```

Comment:
I'm sure @janisozaur will love seeing _another_ heading being added to PCH.... ;)

Reply:
Sorry! 

(I will move this one to the top, just before WinRT Headers)